x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1230,50 +1231,47 @@ private static VersionedApiKeyDoc single(final String apiKeyId, final Collection
     }
 
     private BulkRequest buildBulkRequestForUpdate(
-        final Collection<VersionedApiKeyDoc> currentVersionedDocs,
+        final VersionedApiKeyDoc apiKeyDoc,
```

Comment:
I couldn't resist a small refactor here: we already enforce that we have a single api key doc before calling this method. It makes little sense to go back to suddenty handling a collection.